Rorys branch sprint4

Roguma opened the pull request:
Added code for tests and ran some reformatting in pycharm

Roguma committed:
Update orderHandler.py

fixing a variable name order_status  to new_order_status.

reformatted the file to reflect python coding style

Roguma committed:
formatting

removed unnecessary parentheses from connection\test\test.py
added a white line to orders\test\test.py

Roguma committed:
tests and reformatting

Reformatting of main.py to python coding standards.

writing tests that were stubbed out in
products\tests\test.py

Roguma committed:
Update test.py

Oops forgot the "assert"s for the tests.
Cleaned up some of the unnecessary keys
and variables, as well as table objects.

Roguma committed:
Update test.py

Fixed test_cancel_order :)

Roguma committed:
Merge branch 'discord' of https://github.com/ThePinkPythons/Database-Project into Rorys-Branch-Sprint4

 Conflicts:
	main.py

asevans48 commented:
Please check the main.py file and resolve conflicts. Otherwise, I will see about hooking this into a CI/CD pipeline. Thanks

Roguma committed:
updates 5/4/2021

fixed my main.py to match the head
working on unittests
updated a file name to match for the code.txt

Roguma commented:
I updated the branch, that should have resolved the conflicts.

Roguma committed:
Update test.py

test added for create_order

Roguma committed:
Update test.py

update test_create_order to better reflect the actual create order function

Roguma committed:
Update user_functions.py

remove_user has been deleted as there is no current use for it.

asevans48 commented:
awesome. I will merge and pull into my current working branch to test out the db thing. pretty sure it was the command line issue
